docs/development-guide.md

```diff
@@ -122,13 +122,30 @@ Signed-off-by: Random J Developer <random@developer.example.org>
 The format can be described more formally as follows:
 
 ```text
-<subsystem>: <what changed>
+<component>: <subject of the change>
 <BLANK LINE>
-<why this change was made>
+<paragraph(s) with reason/description>
 <BLANK LINE>
-<footer>
+<signed-off-by>
 ```
 
+The `component` in the subject of the commit message can be one of the following:
+
+* `cephfs`: bugs or enhancements related to CephFS
+* `rbd`: bugs or enhancements related to RBD
+* `doc`: documentation updates
+* `util`: utilities shared between components use `cephfs` or `rbd` if the
+   change is only relevant for one of the type of storage
+* `journal`: any of the journalling functionalities
+* `helm`: deployment changes for the Helm charts
+* `build`: anything related to building Ceph-CSI, the executable or container
+   images
+* `ci`: changes related to the Continuous Integration, or testing
+* `e2e`: end-to-end testing updates
+* `cleanup`: general maintenance and cleanup changes
+* `revert`: undo a commit that was merged by mistake, use of one of the other
```

Comment:
can we need to split this line, `use of one of the other components is in most cases recommended` can be a separate line?

Reply:
I really intended to have a note about `revert`, it should almost never be used. Adding the `use of one of the other components is in most cases recommended` on its own line does not make much sense to me.